packages/perseus-editor/src/widgets/image-editor.tsx

```diff
@@ -277,7 +277,13 @@ const ImageEditor: any = createReactClass({
     // silently load the image when the component mounts
     // silently update url and sizes when the image loads
     // noisily load the image in response to the author changing it
-    onUrlChange: function (url, silent) {
+    onUrlChange: async function (url: string | undefined | null, silent) {
+        // Check if we've been passed something that looks like a URL
+        if (!url) {
+            this.setUrl(url, 0, 0, silent);
```

Comment:
Looking at `setUrl`, if we get a value that's not a valid URL (even `null` or `undefined`), we'll set that value onto the Image.url attribute and the new error handling should handle it?  Just want to make sure we aren't burying a new error here.